app/src/main/java/cs5150athletetracking/com/athletetracking/LocationJSON.java

```diff
@@ -33,13 +37,20 @@ public String toString(){
     private final String TAG = "LocationJSON";
 
     public LocationJSON(String username, String timestamp,
-                        double latitude, double longitutde,
+                        double latitude, double longitude,
                         double altitude){
         try {
+            this.params = new LinkedHashMap<>();
```

Comment:
No point in this being a JSON if we're just going to use a map and use the current uploader strategy. I'm sure we can just upload the JSON string directly without all of this extra work that ultimately limits our JSON structure.